.coveragerc

```diff
@@ -0,0 +1,6 @@
+[report]
+show_missing = True
+fail_under = 90
```

Comment:
Can we aim to 100?

Reply:
The coverage was already at 100%, I changed the constraint.